psi4/driver/procrouting/proc_table.py

```diff
@@ -43,196 +42,197 @@
+        'fno-mp3': proc.run_fnocc,
+        'fno-mp4(sdq)': proc.run_fnocc,
+        'fno-mp4': proc.run_fnocc,
+        'fno-lccd': proc.run_cepa,
+        'fno-lccsd': proc.run_cepa,
+        'fno-cepa(0)': proc.run_cepa,
+        'fno-cepa(1)': proc.run_cepa,
+        'fno-cepa(3)': proc.run_cepa,
+        'fno-acpf': proc.run_cepa,
+        'fno-aqcc': proc.run_cepa,
+        'fno-cisd': proc.run_cepa,
+        'lccd': proc.select_lccd,
+        'lccsd': proc.run_cepa,
+        'cepa(0)': proc.run_cepa,
+        'cepa(1)': proc.run_cepa,
+        'cepa(3)': proc.run_cepa,
+        'acpf': proc.run_cepa,
+        'aqcc': proc.run_cepa,
+        'efp': proc.run_efp,
+        'dmrg-scf': proc.run_dmrgscf,
+        'dmrg-caspt2': proc.run_dmrgscf,
+        'dmrg-ci': proc.run_dmrgci,
+        # Upon adding a method to this list, add it to the docstring in energy() below
+        # Aliases are discouraged. If you must add an alias to this list (e.g.,
+        #    lccsd/cepa(0)), please search the whole driver to find uses of
+        #    name in return values and psi variables and extend the logic to
+        #    encompass the new alias.
+    },
+    'gradient': {
+        'hf': proc.run_scf_gradient,
+        'scf': proc.run_scf_gradient,
+        'cc2': proc.run_ccenergy_gradient,
+        'ccsd': proc.select_ccsd_gradient,
+        'ccsd(t)': proc.select_ccsd_t__gradient,
+        'mp2': proc.select_mp2_gradient,
+        'eom-ccsd': proc.run_eom_cc_gradient,
+        'dcft': proc.run_dcft_gradient,
+        'omp2': proc.select_omp2_gradient,
+        'omp3': proc.select_omp3_gradient,
+        'mp3': proc.select_mp3_gradient,
+        'mp2.5': proc.select_mp2p5_gradient,
+        'omp2.5': proc.select_omp2p5_gradient,
+        'lccd': proc.select_lccd_gradient,
+        'olccd': proc.select_olccd_gradient,
+        'ccd': proc.run_dfocc_gradient,
+        # Upon adding a method to this list, add it to the docstring in optimize() below
+    },
+    'hessian': {
+        # Upon adding a method to this list, add it to the docstring in frequency() below
+        'hf': proc.run_scf_hessian,
+        'scf': proc.run_scf_hessian,
+    },
+    'properties': {
+        'hf': proc.run_scf_property,
+        'scf': proc.run_scf_property,
+        'mp2': proc.select_mp2_property,
+        'cc2': proc.run_cc_property,
+        'ccsd': proc.run_cc_property,
+        'eom-cc2': proc.run_cc_property,
+        'eom-ccsd': proc.run_cc_property,
+        'detci': proc.run_detci_property,  # full control over detci
+        'cisd': proc.run_detci_property,
+        'cisdt': proc.run_detci_property,
+        'cisdtq': proc.run_detci_property,
+        'ci': proc.run_detci_property,  # arbitrary order ci(n)
+        'fci': proc.run_detci_property,
+        'rasscf': proc.run_detci_property,
+        'casscf': proc.run_detci_property,
+        # Upon adding a method to this list, add it to the docstring in property() below
+    }
+}
```

Comment:
I think this is a bit less readable. Add ` # yapf: disable` to the closing dict curly brace will make yapf leave it alone. Revert, I think (or check in unformatted copy and we'll squash the PR).

Reply:
I don't think I'm following. You seem to be to saying that you want the two closing braces on the same line, so I should revert the entire commit, add the yapf disable command to the brace line, repeat the yapf/commit/push, so you can squash. Is that right? If so, why not just have me change the brace and add the yapf disable without reverting the commit?

Reply:
We want all the changes to the procedures dictionary to go away. But while you’re at it, may as well make it invulnerable to future “yapf -i proc_table.py”. Adding the yapf disable on the final dict line will prevent formatting of the whole dict.

Reply:
Does "all the changes" include the fact that yapf wants energy/gradient/properties indented 4 spaces instead of 8? I can't see how keeping all 8 spaces improves readability.

Reply:
All I really want is for the `: proc.` to remain vertically aligned. All the other reformatting yapf has done to the dictionary (incl. 8->4) is good by me. That will require some hand formatting -- I was just trying to offer an easy way out by "revert all".